arch/eventloop_posix_udp.c

```diff
@@ -30,6 +30,8 @@
 /* A registered file descriptor with an additional method pointer */
 typedef struct {
     UA_RegisteredFD fd;
+    struct sockaddr* sendAddr;
```

Comment:
Use `sockaddr_storage` directly instead of a pointer.
That is guaranteed to be large enough for all addresses. And you don't have to malloc.
https://pubs.opengroup.org/onlinepubs/000095399/basedefs/sys/socket.h.html